routes/api.php

```diff
@@ -168,6 +170,9 @@
         // activity-items
         Route::resource('activity-items', 'ActivityItemController');
 
+        // organization-types
+        Route::resource('organization-type', 'OrganizationTypesController');
```

Comment:
'organization-type' -> 'organization-types'

I think it should be apiResource. Please also update other resource routes in admin routes.